CHANGELOG.md

```diff
@@ -7,6 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Coming soon
 * TBD
 
+## Version 7.3.0 (2021-9-30)
+
+### New features and improvements
+
+* Support for [Java 16](https://github.com/newrelic/newrelic-java-agent/pull/366)
```

Comment:
:pray: Do you think #433 will make it into the release as well?

(I tested #366 + #437 + the current state of #433 on top of release 7.2.0, i.c.w. a JDK 17 base image, and it seemed to work fine. Class files did still target Java 11, though.)

Reply:
Doesn't look like we'll be able to get #433 in 7.3.0.  Our current testing is looking good as well, but we have some testing goals and reviews to hit that would make it a rush job to get finished before 7.3.0 releases. We expect 17 support to be pretty quick on the heels of that release though.